content/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/configuring-git-for-github-desktop.md

```diff
@@ -1,62 +1,114 @@
+{% data reusables.desktop.select-email-git-config %}
+  ![Select email address in Git configuration field](/assets/images/help/desktop/mac-email-git-config.png)
 {% data reusables.desktop.click-save-git-config %}
   ![Save button in Git configuration field](/assets/images/help/desktop/mac-save-git-config.png)
 
 {% endmac %}
 
 {% windows %}
 
-{% data reusables.desktop.sign-in-choose-product %}
-{% data reusables.user-settings.access_settings %}
-{% data reusables.user-settings.emails %}
-{% data reusables.desktop.copy-email-git-config %}
-{% data reusables.desktop.return-to-desktop %}
 {% data reusables.desktop.windows-choose-options %}
 8. In the Options window, click **Git**.
 ![The Git Pane in the Options menu](/assets/images/help/desktop/windows-select-git-pane.png)
 {% data reusables.desktop.name-field-git-config %}
   ![The name field of the Git configuration](/assets/images/help/desktop/windows-name-git-config.png)
-{% data reusables.desktop.paste-email-git-config %}
-  ![Pasted email address in Git configuration field](/assets/images/help/desktop/windows-email-git-config.png)
+{% data reusables.desktop.select-email-git-config %}
+  ![Select email address in Git configuration field](/assets/images/help/desktop/windows-email-git-config.png)
 {% data reusables.desktop.click-save-git-config %}
   ![Save button in Git configuration field](/assets/images/help/desktop/windows-save-git-config.png)
 
 {% endwindows %}
 
+## Overriding the email address GitHub Desktop uses for a repository
+
+In {% data variables.product.prodname_desktop %} you can change the name and email address used for commits in a specific repository. This local Git configuration will override your global Git configuration settings in this one repository, but will not affect any other repositories.
+
+{% mac %}
+
+{% data reusables.desktop.mac-repository-settings-menu %}
+{% data reusables.desktop.select-git-config %}
+{% data reusables.desktop.use-local-git-config %}
+{% data reusables.desktop.local-config-name %}
+{% data reusables.desktop.local-config-email %}
+{% data reusables.desktop.repository-settings-save %}
+
+{% endmac %}
+
+{% windows %}
+
+{% data reusables.desktop.windows-repository-settings-menu %}
+{% data reusables.desktop.select-git-config %}
+{% data reusables.desktop.use-local-git-config %}
+{% data reusables.desktop.local-config-name %}
+{% data reusables.desktop.local-config-email %}
+{% data reusables.desktop.repository-settings-save %}
+
+{% endwindows %}
+
+
+## Configuring your default branch for new repositories
+
+You can configure the default branch that will be used when creating a new repository in {% data variables.product.prodname_desktop %}. The default branch is the base branch for pull requests and code commits. For more information about the default branch, see "[About the default branch](/github/collaborating-with-issues-and-pull-requests/about-branches#about-the-default-branch)."
```

Comment:
I think it's misleading to say the default branch is the base branch for all pull requests, since "base branch" has a PR-specific meaning (and can be any branch). Let's just let folks read the article if they don't already know what a default branch it.
```suggestion
You can configure the default branch that will be used when you create a new repository in {% data variables.product.prodname_desktop %}. For more information about the default branch, see "[About the default branch](/github/collaborating-with-issues-and-pull-requests/about-branches#about-the-default-branch)."
```